type/__apt_source/files/source.list.template

```diff
@@ -1,15 +1,9 @@
-#!/bin/sh
+#!/bin/sh -e
 set -u
 
 entry="$uri $distribution $component"
-if test -n "${forcedarch-}"
-then
-   options=${options-}${options:+ }${forcedarch}
-fi
-if test -n "${signed_by-}"
-then
-   options=${options-}${options:+ }${signed_by}
-fi
+options="${forcedarch-} ${signed_by-}"
+options=${options## }; options=${options%% } # Trim spaces from either end
```

Comment:
this looks confusing with `;`, put them on separate lines

Reply:
I find, those two belong together very closely; hence I kept them on a single line.

To be honest, first I thought, I'll go `options=${${options## }%% }`. Which is fine, but does look a bit funny and confusing...

Reply:
> To be honest, first I thought, I'll go `options=${${options## }%% }`

If you add comment before it, then it's fine.